pom.xml

```diff
@@ -19,18 +19,52 @@
 
         <sonar.organization>taragrammatonn</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+        <testcontainers.version>1.19.3</testcontainers.version>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter</artifactId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
How is it used ? You already have postgresql

Reply:
Această dependență facilitează lucrul meu cu clasa CustomerRepository și îmi permite să operez cu metodele CRUD aplicate în CustomerControllerTest.

Reply:
Sorry, i've missed the line